app.js

```diff
@@ -65,6 +66,8 @@ if (currentTheme === undefined) {
   store.setUserPref('app.theme', themes[0].key);
 }
 
+store.set('userPrefs.app.zoomToken', '');
```

Comment:
lets move this to when the app is ready so we don't run into that issue with "new" installs that don't have the preferences files.